mlir/include/mlir/Target/LLVMIR/ModuleImport.h

```diff
@@ -350,7 +355,8 @@ class ModuleImport {
   DictionaryAttr convertParameterAttribute(llvm::AttributeSet llvmParamAttrs,
                                            OpBuilder &builder);
   /// Converts the parameter and result attributes attached to `call` and adds
-  /// them to the `callOp`.
+  /// them to the `callOp`. Implemented in terms of the other definition of
+  /// the public definition of convertParameterAttributes.
```

Comment:
```suggestion
  /// them to the `callOp`. Implemented in terms of the 
  /// the public definition of convertParameterAttributes.
```
nit: I think there was one definition too much. Also feel to drop the sentence since such comments may start to be confusing should we move code around later on. WDYT?

Reply:
Sounds good!